src/features_x86.h

```diff
@@ -108,6 +108,7 @@ JL_FEATURE_DEF(clzero, 32 * 8 + 0, 0)
 JL_FEATURE_DEF(wbnoinvd, 32 * 8 + 9, 0)
 
 // EAX=7,ECX=1: EAX
+JL_FEATURE_DEF(avxvnni, 32 * 9 + 4, 0)
```

Comment:
This is added in LLVM 12 so it should be have that set as the version requirement (i.e. the `0` should be `120000`).